### PR TITLE
Check PAGE_TEMPLATES setting

### PR DESCRIPTION
**Pull request by jpic**

### CODE REVIEW

**Comment by remik** [comment]
You need to replace `basic_cms` to `pages`.

**Comment by jpic** [comment]
Thanks bro don't worry I'm monitoring travis on this ;)

File: doc/settings-list.rst [hunk 1]
```diff
@@ -40,6 +40,11 @@ Where the model might look like this::
         def __unicode__(self):
             return self.name
 
+.. warning:: basic_cms checks that every template in PAGE_TEMPLATE exists with
```
Comment (batiste) [inline doc/settings-list.rst]: Just change basic_cms to "this CMS", "Django Page CMS" or "Gerbi CMS"

**Comment by batiste** [comment]
Apart from the little comment this all good!

**Comment by jpic** [comment]
Updated

**Comment by jpic** [comment]
Updated, thanks for reviewing !

**Pull request merged**